REQUIRE

```diff
@@ -1,3 +1,3 @@
 julia 0.5
-OpenCL
+OpenCL 0.5.2
```

Comment:
By making bounds stricter than they absolutely need to be, you're actually more likely to prevent anyone who happens to have OpenCL 0.5.0 or 0.5.1 pinned from getting the latest version of CLBLAS. Pkg will just hold them back, but those version combinations should work together, shouldn't they?